.github/CONTRIBUTING.md

```diff
@@ -89,8 +89,8 @@ Once you know what the first small piece of your feature will be, follow this ge
   - If there are visual changes, add before and after screenshots in the pull request comments.
   - Add unit tests, or at a minimum, provide helpful instructions for the reviewer so they can test your changes. This will help speed up the review process.
   - Ensure that your commit messages are [meaningful](http://robots.thoughtbot.com/5-useful-tips-for-a-better-commit-message).
-6. Mention that the PR is ready for review or if you have write access remove the **<span class="label status-in-progress">[Status] In Progress</span>** label from the pull request and add the **<span class="label status-needs-review">[Status] Needs Review</span>** label - someone will provide feedback on the latest unreviewed changes. The reviewer will also mark the pull request as **<span class="label status-awaiting-fixes">[Status] Awaiting Fixes</span>** if they think you need to change anything.
-7. If you get a <img src="https://assets-cdn.github.com/images/icons/emoji/unicode/1f44d.png" class="emoji" />, <img src="https://assets-cdn.github.com/images/icons/emoji/unicode/1f4a5.png" class="emoji" />, <img src="https://assets-cdn.github.com/images/icons/emoji/unicode/1f6a2.png" class="emoji" />, <img src="https://assets-cdn.github.com/images/icons/emoji/shipit.png" class="emoji" />, or a LGTM and the status has been changed to **<span class="label status-ready-to-merge">[Status] Ready to Merge</span>** – this is great – the pull request is ready to be merged into `master`.
+6. Mention that the PR is ready for review or if you have write access remove the **<span class="label status-in-progress">[Status] In Progress</span>** label from the pull request and add the **<span class="label status-needs-review">[Status] Needs Review</span>** label - someone will provide feedback on the latest unreviewed changes. The reviewer will also mark the pull request as **<span class="label status-needs-author-reply">[Status] Needs Author Reply</span>** if they think you need to change anything.
+7. If you get a <img src="https://github.githubassets.com/images/icons/emoji/unicode/1f44d.png" class="emoji" />, <img src="https://github.githubassets.com/images/icons/emoji/unicode/1f4a5.png" class="emoji" />, <img src="https://github.githubassets.com/images/icons/emoji/unicode/1f6a2.png" class="emoji" />, <img src="https://github.githubassets.com/images/icons/emoji/shipit.png" class="emoji" />, or a LGTM and the status has been changed to **<span class="label status-ready-to-merge">[Status] Ready to Merge</span>** – this is great – the pull request is ready to be merged into `master`.
```

Comment:
Wouldn't just actual emojis work here since these are standards? (apart from maybe _shipit_, the rest 👍💥🚢)

Also using `:thumbsup: :boom: :ship: :shipit:` might work?

Reply:
👍 

Switched to the actual emojis except for shipit (replaced with `:shipit`:)